app/screens/navigation.ts

```diff
@@ -421,6 +421,7 @@ export async function dismissAllModalsAndPopToScreen(screenId: string, title: st
         }
         try {
             await Navigation.popTo(screenId, mergeOptions);
+            await Navigation.updateProps(screenId, passProps);
```

Comment:
added

Reply:
This looks ok, just wondering if this should be optional so that if there is no need to update the props and this function is called we could then avoid unnecessary re-renders

Reply:
Ah, good point. Fixed -- will call update props if we're given passProps.